app/Http/Controllers/Articles/ArticlesController.php

```diff
@@ -47,9 +47,10 @@ public function show(Article $article)
             404,
         );
 
-        $trendingArticles = Cache::remember('trendingArticles', now()->addHour(), function () {
+        $trendingArticles = Cache::remember('trendingArticles', now()->addHour(), function () use ($article) {
             return Article::published()
                 ->trending()
+                ->where('id', '!=', $article->id)
```

Comment:
```suggestion
                ->whereKeyNot($article)
```

Reply:
Ahh nice, I didn't actually know you could do that. I've added that now :)